tools/proofers/proof_frame_std.inc

```diff
@@ -1,37 +1,95 @@
 <?php
 include_once($relPath.'http_headers.inc');
 include_once($relPath.'slim_header.inc');
+include_once("proof_image.inc"); // get_image_data()
+include_once("std_view.inc");
 
 function echo_proof_frame_std($ppage)
 {
-    $user = User::load_current();
-
-    if ($user->profile->i_layout == '1') {
-        $rows_or_cols = "COLS='*,{$user->profile->v_tframe}%'";
-    } else {
-        $rows_or_cols = "ROWS='*,{$user->profile->h_tframe}%'";
-    }
-
-    $params = $ppage->urlencoded(true);
-
-    if (($user->profile->i_layout == '1' && $user->profile->v_tscroll == 0)
-        ||
-         ($user->profile->i_layout == '0' && $user->profile->h_tscroll == 0)) {
-        $text_scrolling = "SCROLLING='no'";
-    } else {
-        $text_scrolling = "";
-    }
-
-    $margins = "marginwidth='2' marginheight='2'";
-
-    // -------------------------------------
-
-    slim_header_frameset($ppage->projectid());
-    echo "
-        <FRAMESET $rows_or_cols>
-        <FRAME NAME='imageframe' SRC='image_frame_std.php?$params' $margins>
-        <FRAME NAME='textframe' SRC='text_frame_std.php?$params' $margins $text_scrolling>
-        </FRAMESET>
-        <NOFRAMES>"._("Your browser is not frames capable!")."</NOFRAMES>
-    ";
+    global $code_url;
+
+    $project = new Project($ppage->projectid());
+    $valid_character_pattern = build_character_regex_filter($project->get_valid_codepoints(), "js");
+    $switch_confirm = json_encode(_('Are you sure you want to save the current page and change layout?'));
+    $revert_confirm = json_encode(_("Are you sure you want to save the current page and revert to the original text for this round?"));
+
+    $header_args = [
+        "js_files" => [
+            "$code_url/scripts/splitControl.js",
+            "proof_std.js",
```

Comment:
no prefix like the others?

Reply:
This is inconsistent. In some cases there is prefix even though it is in the same directory. Probably best to remove them in that case.